community/kernel/src/main/java/org/neo4j/kernel/impl/util/CappedLogger.java

```diff
@@ -53,6 +54,14 @@ public CappedLogger( @Nonnull Log delegate )
         this.delegate = delegate;
     }
 
+    public void debug( @Nonnull String msg )
+    {
+        if ( filter.accept( msg, null ) )
+        {
+            delegate.debug( msg );
```

Comment:
Why is null ok in `filter.accept` and not here? Because the Log interface/superclass defines the methods as NonNull.